ReportPortal.Client.Tests/LaunchItem/LaunchItem.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace ReportPortal.Client.Tests.LaunchItem
 {
+    [Trait("LaunchItem", "")]
```

Comment:
Incorrect usage of traits. Need smt like `Trait("Endpoint", "Launches")`. The 1st argument is category name.